gtsam/navigation/NavState.h

```diff
@@ -83,6 +83,14 @@ class GTSAM_EXPORT NavState {
   const Point3& position(OptionalJacobian<3, 9> H = boost::none) const;
   const Velocity3& velocity(OptionalJacobian<3, 9> H = boost::none) const;
 
+  ///< Syntactic sugar
```

Comment:
Harmful sugar as no meaningful math equivalent that anyone would intuitively agree to.